src/resolvers/accounts.ts

```diff
@@ -0,0 +1,89 @@
+import { ResolverContextBase } from '../types/graphql';
+import { Account, AccountType } from '../models/account';
+import { Currency } from '../types/currency';
+
+/**
+ * Concrete create mutation input declaration
+ */
+interface CreateMutationInput {
```

Comment:
может лучше

```suggestion
interface AccountCreateMutationInput {
```

или

```suggestion
interface CreateAccountMutationInput {
```

Reply:
done